doc/source/release-notes/Percona-Server-5.5.46-37.5.rst

```diff
@@ -0,0 +1,45 @@
+
+New Features
+============
+
+ |Percona Server| is now available for *Ubuntu* 15.10 (Wily).
+
+Bugs Fixed
+==========
+
+ Manipulating the :variable:`innodb_track_redo_log_now` variable dynamically would crash the server if it was started without :variable:`innodb_track_changed_pages` enabled. This variable is available on debug builds only. Bug fixed :bug:`1368530`.
+
+ A potential crash in handling corrupted tables with :variable:`innodb_corrupt_table_action` ``warn`` or ``salvage`` values has been fixed. Bug fixed :bug:`1426610`.
+
+ If the user had duplicate pid-file options in config files when running ``yum upgrade``, the upgrade would stop with error because it would think it found the duplicate pid while it was the same pid specified twice.  Bug fixed :bug:`1454917`.
+
+ On some filesystems server would not start if :ref:`changed_page_tracking` feature was enabled and :variable:`innodb_flush_method` variable was set to ``O_DIRECT``. Bugs fixed :bug:`1500720` and :bug:`1498891`.
+
+ When :ref:`user_stats` are enabled, after executing any statement of the ``SHOW ...`` family that produces non-empty result, the total number of the sent rows is updated in the :table:`USER_STATISTICS` ``ROWS_FETCHED`` column. Bug fixed :bug:`1510953`.
+
+ Fixed the conflicting meta packages between 5.1, 5.5, and 5.6 release series in *Debian* and *Ubuntu* distributions. ``percona-server-server`` and ``percona-server-client`` meta packages now point to the latest 5.6 release. Bug fixed :bug:`1292517`.
+
+ :table:`INNODB_CHANGED_PAGES` table was unavailable with non-default :variable:`innodb_data_home_dir` setting if the variable had a trailing slash. Bug fixed :bug:`1364315`.
+
+ ``UPDATE`` statement could crash the server with :ref:`innodb_fake_changes_page` enabled. Bug fixed :bug:`1395706`. 
+
+ When :ref:`innodb_fake_changes_page` feature is enabled database records are not actually modified but the modification is assumed and accordingly the path is selected/forced. For workload involving ordering change ``UPDATE`` will be executed as series of ``DELETE`` and ``INSERT`` action. Given that fake-change mode is enabled actual delete will never happen but path will be selected assuming that it took place. This can cause server to hit some asserts in the forced path as the state was not marked. One such path that assumed record is marked as deleted when it was not is now fixed. Bug fixed :bug:`1395706`.
+
+ Changing :variable:`innodb_fake_changes` variable value in the middle of transaction would have an immediate effect, that is, making part of transaction run with fake changes enabled and the rest with fake changes disabled, resulting in broken transaction. Fixed by making any :variable:`innodb_fake_changes` value changes becoming effective at the start of next transaction instead of next statement. Bug fixed :bug:`1395579`.
```

Comment:
in the middle of a transaction
making part of the transaction
in a broken transaction
of the next transaction
of the next statement